mlflow_extensions/serving/engines/huggingface_utils.py

```diff
@@ -21,6 +23,47 @@ def snapshot_download_local(repo_id: str, local_dir: str, tokenizer_only: bool =
     return model_local_path
 
 
+def get_local_snapshot(repo_id, cache_dir, local_dir):
```

Comment:
can we make this type hinted and forced to be kwargs, so that the receiver passes the right args in the right order 

def get_local_snapshot(*, repo_id: str, cache_dir: str, local_dir: str):

cache_dir can also be None so if cache_dir is None we should skip alot of this logic right?